employee.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Employees</title>
+    <link rel="stylesheet" href="main.css">
+</head>
+<body>
+    <h1 class="about-header">About Us</h1>
+    <p>Marzipan cheesecake cheesecake powder oat cake biscuit. Croissant gummies ice cream sugar plum icing jujubes. Cookie marshmallow candy canes jelly beans donut chocolate bar I love. Topping danish donut jelly beans sugar plum apple pie. Gummi bears candy ice cream. Sugar plum sesame snaps cupcake. Gummies jujubes tiramisu halvah oat cake halvah. Marshmallow sesame snaps biscuit I love caramels I love topping bear claw. Biscuit I love cotton candy sesame snaps I love croissant soufflé sugar plum. Ice cream apple pie dragée jujubes powder dragée cupcake carrot cake.</p>
```

Comment:
Cheesecake is repeated in this mission statement.

Reply:
don't we want more than one cheesecake between the three of us?